modules/involveApi.js

```diff
@@ -0,0 +1,13 @@
+// https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/X2HxEwPkDolJVV1WCgVc
+const createApiId = async () => {
+  /* const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const json = await response.text();
+  console.log(`New app created with id: ${json}`); */
+};
+
```

Comment:
- [ ] You have commented code in this file, even this file is useless because there is nothing in this file other than having an empty function. It would be nice if you remove it to decrease the size of your project which will help your web application load and run faster :blush: